x/posts/internal/types/genesis.go

```diff
@@ -1,17 +1,21 @@
 package types
 
+import "fmt"
+
 // GenesisState contains the data of the genesis state for the posts module
 type GenesisState struct {
 	Posts       Posts                  `json:"posts"`
 	PollAnswers map[string]UserAnswers `json:"poll_answers_details"`
 	Reactions   map[string]Reactions   `json:"reactions"`
+	Hashtags    map[string]PostIDs     `json:"hashtags"`
```

Comment:
I think this can be removed from the genesis file as we are now reading this directly from the posts messages. Am I right?